language-agent/JVMMetric.proto

```diff
@@ -85,8 +86,22 @@ enum GCPhrase {
 }
 
 message Thread {
-  int64 liveCount = 1;
-  int64 daemonCount = 2;
-  int64 peakCount = 3;
+    int64 liveCount = 1;
+    int64 daemonCount = 2;
+    int64 peakCount = 3;
+    int64 deadlocked = 4;
+    int64 monitorDeadlocked = 5;
+    int64 newThreadCount = 6;
+    int64 runnableThreadCount = 7;
+    int64 blockedThreadCount = 8;
+    int64 waitThreadCount = 9;
+    int64 timeWaitThreadCount = 10;
```

Comment:
```suggestion
    int64 waitingThreadCount = 9;
    int64 timedWaitingThreadCount = 10;
```
I think it would be better to name like this.

Reply:
resolve, please review it.